packages/kit/src/runtime/server/page/load_data.spec.js

```diff
@@ -51,12 +50,15 @@ test('errors when no acao header present on cors', async () => {
 	try {
 		const response = await fetch('https://domain-b.com');
 		await response.text();
-		assert.unreachable('should have thrown cors error');
+		throw new Error('should have thrown cors error');
 	} catch (e) {
-		assert.ok(e instanceof Error);
-		assert.match(
-			e.message,
-			/CORS error: No 'Access-Control-Allow-Origin' header is present on the requested resource/
+		if (!(e instanceof Error)) {
+			throw new Error(`Expected an Error to be thrown but received ${typeof e}`);
+		}
+		assert.isTrue(
+			e.message.includes(
+				"CORS error: No 'Access-Control-Allow-Origin' header is present on the requested resource"
+			)
 		);
 	}
```

Comment:
I think this can be replaced with a expect.rejects.toThrowError

https://vitest.dev/api/expect.html#tothrowerror

Reply:
nice, updated